chpl/chpl-service/src/main/java/gov/healthit/chpl/search/ListingSearchService.java

```diff
@@ -84,6 +84,7 @@ public ListingSearchResponse findListings(SearchRequest searchRequest) throws Va
             .filter(listing -> matchesDerivedCertificationEditions(listing, searchRequest.getDerivedCertificationEditions()))
             .filter(listing -> matchesCertificationEditions(listing, searchRequest.getCertificationEditions()))
             .filter(listing -> matchesDeveloper(listing, searchRequest.getDeveloper()))
+            .filter(listing -> matchesDeveloperId(listing, searchRequest.getDeveloperId()))
```

Comment:
I have lots and lots of unit tests for this class. Can you please add a couple to test this new developerId filtering functionality?